### PR TITLE
fix notion pagination issue

### PR DESCRIPTION
**Pull request by julia-script**
# 🧙 Description

We were only fetching the first page of the notion query, now it fetches all of them

Ticket #

## ✅ Checklist

- [ ] Acceptance Criteria described in the ticket are met
- [ ] Texts and strings are in the related i18n file
- [ ] Utilities have unit tests
- [ ] User stories and functionalities have integration or e2e tests
- [ ] If adding a new token or network
  - [ ] Registry package is updated
  - [ ] Asset images are added
- [ ] Related packages.json are upgraded
- [ ] Changelog is updated

### CODE REVIEW

File: pages/dappstore/src/lib/fetch-dapps.ts [hunk 1]
```diff
@@ -5,19 +5,27 @@ import { ECOSYSTEM_PAGE_NOTION_ID } from "@evmosapps/evmos-wallet/src/internal/w
...
 export const fetchDapps = cache(async () => {
   const dapps = await fetchNotionEcosystemDb();
 
   const parsedDapps = await Promise.all(
-    dapps.results.map(async (value) => {
+    dapps.map(async (value) => {
       const result = await dappSchema.safeParseAsync(value);
+
       if (!result.success) {
+        debugger;
```
Comment (miliPaoletti) [inline pages/dappstore/src/lib/fetch-dapps.ts:28]: Shouldn't we remove this debugger?

File: pages/dappstore/src/pages/landing/Test.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+"use client";
```
Comment (miliPaoletti) [inline pages/dappstore/src/pages/landing/Test.tsx:1]: Is this relevant to this PR ?

**Review by miliPaoletti — APPROVED** [review]
Looks good! Left a few comments! 🚀

File: pages/dappstore/src/pages/landing/Test.tsx (hunk 2 above)
Comment (julia-script) [reply pages/dappstore/src/pages/landing/Test.tsx:1]: no, lol
